src/top-navigation/styles.scss

```diff
@@ -230,25 +230,26 @@
   border-block-end: awsui.$border-divider-section-width solid awsui.$color-border-divider-default;
 
   &-text {
-    @include styles.font-panel-header;
     color: awsui.$color-text-heading-default;
     flex: 1;
     margin-block: 0;
     margin-inline: 0;
     text-align: center;
 
+    &--title {
+      /* used in test-utils */
```

Comment:
Originally we put these comments to prevent SASS removing empty rules during compilation. 

In this case, it is not really needed, but you can keep it if it helps with readaibility

Reply:
Yes, I decided to keep here because it seems like we don't have good coverage for the test util that uses this selector, so I want to make sure it is not removed in future refactors.